benchmarks/models_config/bert_cpu.yaml

```diff
@@ -0,0 +1,21 @@
+---
+bert:
```

Comment:
In the README create a page with the format and clarify all the available parameters values for example gpu takes all and which ones accept lists as input

Reply:
And generally we can add explanation to let users how auto_benchmark and benchmar-ab differ, also the type of inputs they get.